src/main/java/cope/cosmos/client/features/command/commands/SetCommand.java

```diff
@@ -0,0 +1,88 @@
+package cope.cosmos.client.features.command.commands;
+
+import com.mojang.brigadier.arguments.StringArgumentType;
+import com.mojang.brigadier.builder.LiteralArgumentBuilder;
+import com.mojang.brigadier.builder.RequiredArgumentBuilder;
+import com.mojang.brigadier.context.CommandContext;
+import com.mojang.brigadier.suggestion.Suggestions;
+import com.mojang.brigadier.suggestion.SuggestionsBuilder;
+import com.mojang.realmsclient.gui.ChatFormatting;
+import cope.cosmos.client.Cosmos;
+import cope.cosmos.client.features.command.Command;
+import cope.cosmos.client.features.modules.Module;
+import cope.cosmos.client.features.setting.Setting;
+
+import java.awt.*;
+import java.util.concurrent.CompletableFuture;
+
+public class SetCommand extends Command {
+
+    public SetCommand() {
+        super("Set", "Sets a setting in a module", LiteralArgumentBuilder.literal("set")
+                .then(RequiredArgumentBuilder.argument("module", StringArgumentType.string()).suggests((context, builder) -> suggestNames(builder))
+                        .then(RequiredArgumentBuilder.argument("setting", StringArgumentType.string()).suggests((context, builder) -> suggestSettingNames(builder, context))
+                                .then(RequiredArgumentBuilder.argument("value", StringArgumentType.string()).suggests((context, builder) -> suggestValues(builder, context))
+                                    .executes(context -> {
+                                        Module m = Cosmos.INSTANCE.getModuleManager().getModule(module -> module.getName().equalsIgnoreCase(StringArgumentType.getString(context, "module")));
+                                        Setting setting = m.getSettings().stream().filter(s -> s.getName().equalsIgnoreCase(StringArgumentType.getString(context, "setting"))).findFirst().orElse(null);
```

Comment:
make it `Setting<?>` rather than `Setting` because that produces a w*rning and fuck w*rnings